### DISCUSSION TITLE
test

### DISCUSSION THREAD

**Discussion by zkitefly** [zkitefly/zkitefly.github.io#7] [Announcements]
> ##  前言
> 由于 PCL2 仓库中反馈数量较多，因此在反馈者及 Collaborators 进行反馈搜索与查重时可能会搜索不到相关已提交的反馈。此处我从 Bilibili 更新日志评论区及 Github 仓库的 Issue、Discussions 中整理出了一份常见反馈列表，均为常见问题与高频重复反馈，以方便诸位 Collaborators 同志们可以更方便地进行查找，同时在这份表格中也给出了相应的解释。
> 
> 毕竟是我一个人整理的，所以必然会有疏漏，如果大家觉得这个想法还不错也可以在下方补充一些遗漏的常见的反馈，非常非常感谢全体社区玩家的贡献！partying_face
> 
> ##  常见反馈及问题列表
> 这个表格大概按照有关问题的重复频率和所属类别简单进行了排序，你可以使用 Ctrl+F 进行快速搜索。暂缓处理的相关问题我会将原始 Issue 和 Discussions 中的投票页面一同展示出来，后者使用括号括起来方便辨认wink
> 
> 除此之外，你还可以在 PCL2 内置帮助库 或 [PCL2 常见问题 兼 脑溢血急救中心](https://shimo.im/docs/qKPttVvXKqPD8YDC) 中找到一些没在这里提及的问题~
> 
> 常见问题	相关解释	参考
> **联机！**	_**“先等到 8 月看看…… 没法急” ——龙腾猫跃**_	**[2.6.2 更新日志](https://www.bilibili.com/read/cv23979077)**
> **1.16.4 及 1.16.5 离线模式无法进入多人游戏界面**	**PCL2 不会解决该问题，这超出了启动器功能范畴。请自行断网启动游戏或安装 [MultiOfflineFix](https://github.com/MCTeamPotato/MultiOfflineFix) 或 [Offline Multiplayer](https://github.com/ChickenPige0n/offline-multiplayer-fabric) 模组**	**#2003 #2017**
> PCL2 消失或报毒	是 Windows Defender 和杀软 的锅，请在 [Windows 安全中心和杀软中添加排除项或白名单](https://support.microsoft.com/zh-cn/windows/%E5%B0%86%E6%8E%92%E9%99%A4%E9%A1%B9%E6%B7%BB%E5%8A%A0%E5%88%B0-windows-%E5%AE%89%E5%85%A8%E4%B8%AD%E5%BF%83-811816c0-4dfd-af4a-47e4-c301afe13b26)。	#56 [#666](https://github.com/Hex-Dragon/PCL2/issues/666#issuecomment-1611534587) PCL2 内置帮助库（等待更新）
> 无法登录正版账户：你可能没有在 Minecraft 官网创建档案，或者没有购买 Minecraft。	是 Mojang 的 Bug，与 PCL2 无关。请前往 [Minecraft.net](https://www.minecraft.net/login) 登录您的 Minecraft 账号后，稍等一会即可使用 PCL2 正常进行登录。	#1299 #1264
> 增加导出整合包功能	暂缓处理。	#98 (#100)
> 支持启动基岩版及 Minecraft 其他衍生游戏	_“这玩意儿我根本启动不了！用官方启动器吧！告辞！” ——龙腾猫跃_	#81
> 支持更新 Mod	暂缓处理。	#102 (#103)
> 爱发电无法收到加群信息或更新密钥	加群请**在爱发电**回复【加群】，更新密钥**在爱发电**获取请回复【更新+识别码】。如无法收到回复，很可能是你已经触发了人工回复，在等待人工回复的过程中无法使用命令。你如果只是想使用命令，可以发送【取消人工处理】。如仍旧没有回复，可以等待一会后再进行尝试。	[石墨文档](https://shimo.im/docs/qKPttVvXKqPD8YDC) #1867 #1529
> 支持 MacOS、Linux、IOS、安卓 等其他操作系统	由于 PCL2 的底层界面框架（WPF）无法跨系统，要兼容其他系统无异于完全重做，不会制作。	#54
> 添加皮肤库	-	#343
> 支持离线披风	技术上无法实现。	#700
> 支持调整 Optifine 和第三方披风选项	暂时不打算制作。	#2045 #2065
> 1.19.3 不支持自定义皮肤	暂缓处理，技术原因。	#1288 (#1596)
> Minecraft 更新后 PCL2 内未找到该最新版本	请在 设置-启动器-下载-版本列表获取 选择 尽量使用官方源。	#1226
> 支持启动器多语言	因涉及页面重新布局、下载源国际适配、第三方网站国际适配、翻译等问题，暂时不计划制作。	#119
> 支持下载地图	存在语言问题。	#44 #135
> 支持下载资源包（材质包&纹理包）	暂缓处理。	#44 (#1095)
> 支持下载第三方服务端及插件	PCL2 是启动器，超出范畴。	#108
> 支持光影下载	没有可用的下载源。	#396
> 支持数据包下载	无可用 API。	#871
> 引进新 Mod / 整合包下载源	没有其他可用的国内的 API。	#1658
> 修改、添加、移除、更新已安装版本的 optifine、forge 等组件和模组加载器	暂缓处理。	#220 (#227)
> 允许管理存档、光影文件	-	#388 #396
> 添加 Mod 收藏夹	暂缓处理。	#487 (#1646)
> 兼容暗黑模式	_“这个基本上要重新设计所有界面，工作量太大了，我猜我都不会做……” ——龙腾猫跃_	#131
> 支持识别、安装 Quilt	暂缓处理。	#48 (#49)
> 添加云端账户、启动器内置社区、云端同步账户数据、官网等	无法抵御攻击，不会制作任何有关云端账号的内容。	#315 等 Issues
> 隐藏主题迁移	考虑到隐私问题，不考虑支持导出此类信息。	#760
> 有关拓展隐藏主题、自定义主题、背景图片功能的 Issue	备注：此类 Issue 大概率会因为必要性及优先级较低被拒绝，但不排除个别会被采纳，仅作参考。	#2216 #1487 #1653 #1242 #840 #791 #711 #562 #401 #131
> 启动器字体自定义	会导致严重的 UI 适配问题，不考虑制作。	#1724 #766 #366
> 无法准确设置游戏分辨率	PCL2 无法准确设置该数值，因为不同系统环境下窗口标题栏和边框的大小不一致。	#1748 #1077
> 有关与 Minecraft 服务器交互的内容	-	#829 #1993
> 允许使用相对路径	_"改这个那一大堆路径设置绝壁一周之内就会出来一大堆反馈让我加相对路径，想想都吓人，算了……" ——龙腾猫跃_	#1568 #1292 #1080
> 将更新日志连接更换为 Biligame Wiki	镜像源更新速度较慢，不考虑使用。	#1884 #1537
> 支持第三方登录全局设置	暂缓处理。	#57 (#58)
> 有关 “千万别点” 的 Issue	_LTCatt added `忽略` and removed `待处理` labels_	#460 #398 #407 #677 #796 #1147 #1923
> 添加更多远古版本	不加.jpg	#698
> 选中 PCL2 文件并快速敲击 Enter 会导致 PCL2 多开并报错提示	无法正确检测，不会修复。	#1372
> 将文件拽进 PCL2 窗口时指针显示禁止符号	检查是否使用了管理员身份启动 PCL2，若确认没有，打开 UAC 再进行尝试	[#2197](https://github.com/Hex-Dragon/PCL2/issues/2197#issuecomment-1626888928)
> 切换音频设备后，音乐播放不会自动切换到新设备	NAudio 的 Bug，技术上无法处理。	#480
> Mod 管理允许扫描子文件夹 Mod	此功能已被移除，由于 .minecraft/mods 文件夹下创建的子文件夹内的 Mod 不被模组加载器（此为如 Forge、Fabric 类模组加载器行为，非 PCL2 行为）加载，因此在新版本中移除了此功能。但如果子文件夹名称为版本名则仍会读取并加载。但是对于 Quilt 加载器下识别子文件夹 Mod 功能暂缓处理。	#811 #1098 #1536 #1784 (#1790)
> ###### **特别鸣谢为这个表格内容与完善工作做出贡献的朋友们：@Chiloven945 @JingHai-Lingyun @WorldHim @SuoHouDaiShi @bwtx2023 @youzi-2333** ；最近更新：2023-07-13

**Comment by WorldHim** [comment]
额，这个好像对表格、超链接和表情的适配都不太好。。

**Reply by WorldHim** [reply]
所以我说他的脚本适配不太好啊（
